sdk/src/client.ts

```diff
@@ -1042,17 +1113,18 @@ export class VaultClient {
 }
 
 const createAta = (
-    wallet: anchor.Wallet,
+    pubkey: PublicKey,
```

Comment:
Can you rename to `owner` or something more descriptive?

Reply:
Done